src/lando/ui/jinja2/partials/job.html

```diff
@@ -21,7 +22,7 @@
                 </p>
             {% endif %}
         {% endif %}
-        {% if job.revisions.exists() and not job.is_pull_request_job %}
+        {% if has_revisions %}
```

Comment:
There is a change in logic here, not sure if it is intentional but this will cause things to load when they weren't before.